provider/datasource_rediscloud_active_active_database.go

```diff
@@ -33,7 +34,12 @@ func dataSourceRedisCloudActiveActiveDatabase() *schema.Resource {
 				Optional:    true,
 			},
 			"memory_limit_in_gb": {
-				Description: "Maximum memory usage for this specific database",
+				Description: "(Deprecated) Maximum memory usage for this specific database",
+				Type:        schema.TypeFloat,
+				Computed:    true,
+			},
+			"dataset_size_in_gb": {
```

Comment:
This attribute could do with a line in `docs/datasources/rediscloud_active_active_subscription_database.md`